src/mui/button/BulkDeleteButton.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import compose from 'recompose/compose';
+import FlatButton from 'material-ui/FlatButton';
+import DeleteIcon from 'material-ui/svg-icons/action/delete';
+import translate from '../../i18n/translate';
+import { crudDelete } from '../../actions/dataActions';
+import { unsetResourcesSelection } from '../../actions/bulkActions';
+
+const BulkDeleteButton = ({
+    label = 'aor.action.delete',
+    translate,
+    resource,
+    selection,
+    crudDelete,
+    unsetResourcesSelection,
+}) => (
+    <FlatButton
+        secondary
+        label={label && translate(label)}
+        icon={<DeleteIcon />}
+        style={{ overflow: 'inherit' }}
+        disabled={!selection.length}
+        onClick={() => {
```

Comment:
Avoid declaring functions inside JSX. Use the bindings inside `connect` to hide the implementation details from the button. It should only know about a `handleClick` function.